### ISSUE TITLE
Xcode 12.5 beta: Nimble linking fails due to now-missing libswiftXCTest

### ISSUE THREAD

**Issue by nikolaykasyanov** [Quick/Nimble#855]
- [x] I have read [CONTRIBUTING](https://github.com/Quick/Nimble/blob/master/CONTRIBUTING.md) and have done my best to follow them.

### What did you do?

Tried Xcode 12.5 beta.

### What did you expect to happen?

I expected Nimble to continue building.

### What actually happened instead?

Build is failing due to missing `libswiftXCTest`. According to the beta [release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-12_5-beta-release-notes/), it's been replaced.

### Environment

List the software versions you're using:

- Quick: *3.0.0*
- Nimble: *9.0.0*
- Xcode Version: *12.5 beta (12E5220o)*
- Swift Version: *5.4*

Please also mention which package manager you used and its version. Delete the
other package managers in this list:

- Cocoapods: *1.10.1*

### Project that demonstrates the issue

Any project out there with with Nimble added via CocoaPods.

**Comment by ikesyo** [comment]
Thanks for raising an issue. Looks like the `libXCTestSwiftSupport` was introduced on Xcode 11 so we can safely replace the references.

**Comment by CraigSiemens** [comment]
Is there a reason the PR from @evelynccordner was closed? It seems like #863 is all that's needed.

**Comment by evelynccordner** [comment]
> Is there a reason the PR from @evelynccordner was closed? It seems like #863 is all that's needed.

@CraigSiemens I wanted to test it on a forked repo first, thanks for the reminder, this fix is working for us.

**Comment by ikesyo** [comment]
resolved by #863.

**Comment by lordzsolt** [comment]
@ikesyo Could you make a numbered release out of it, please?

**Comment by pmanuelli** [comment]
Hello. Will be coming out soon a new version with the fix?

**Comment by khramtsoff** [comment]
Temp workaround
```ruby
post_install do |installer|
    installer.pods_project.targets.each do |target|
        if target.name == "Nimble"
            target.build_configurations.each do |config|
                xcconfig_path = config.base_configuration_reference.real_path
                xcconfig = File.read(xcconfig_path)
                new_xcconfig = xcconfig.sub('lswiftXCTest', 'lXCTestSwiftSupport')
                File.open(xcconfig_path, "w") { |file| file << new_xcconfig }
            end
        end
    end
end
```

**Comment by hbtpoprock** [comment]
> Temp workaround
> 
> ```ruby
> post_install do |installer|
>     installer.pods_project.targets.each do |target|
>         if target.name == "Nimble"
>             target.build_configurations.each do |config|
>                 xcconfig_path = config.base_configuration_reference.real_path
>                 xcconfig = File.read(xcconfig_path)
>                 new_xcconfig = xcconfig.sub('lswiftXCTest', 'lXCTestSwiftSupport')
>                 File.open(xcconfig_path, "w") { |file| file << new_xcconfig }
>             end
>         end
>     end
> end
> ```

@khramtsoff where do I have to put this code in?

**Comment by ikesyo** [comment]
@khramtsoff @hbtpoprock

This is fixed in v9.0.1 or later, so you don't need that workaround anymore.

**Comment by emilianoeloi** [comment]
I need to do this as my app has N targets.
```Ruby
post_install do |installer|
    if target.name == "Nimble"
        target.build_configurations.each do |config|
            base_configuration = config.base_configuration_reference
            if base_configuration && base_configuration.real_path
              xcconfig_path = base_configuration.real_path
              xcconfig = File.read(xcconfig_path)
              new_xcconfig = xcconfig.sub('lswiftXCTest', 'lXCTestSwiftSupport')
              File.open(xcconfig_path, "w") { |file| file << new_xcconfig }
            end
        end
    end
  end
end
```
